src/fastoad/notebooks/02_explanation/04_OpenMDAO.ipynb

```diff
@@ -0,0 +1,892 @@
+{
```

Comment:
<p>1.</p><p>OpenMDAO recommends to use <code>import openmdao.api as om</code>, and we should definitely apply it. This has to be modified in notebooks and python files.</p><p> </p><p>2.</p><blockquote>The docstring has also been added for clarity.</blockquote><p>Is this sentence necessary ? (especially since all previous Python elements in the tutorial have been rightfully written with docstrings)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fast-aircraft-design/FAST-OAD/pull/426/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
1. Done in commit dfacc3b9
2. Agreed, deleted in commit dfacc3b9